backend/src/main/java/model/DataModel.java

```diff
@@ -0,0 +1,7 @@
+package model;
+/**
+ * Created by ciff on 13/10/2017.
+ */
+public interface DataModel<T> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') T is not used in the interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)